pkg/common/cns-lib/vsphere/utils.go

```diff
@@ -48,6 +48,19 @@ func IsNotFoundError(err error) bool {
 	return isNotFoundError
 }
 
+// IsAlreadyExists checks if err is the AlreadyExists fault, if yes then returns true else return false
```

Comment:
This function is also returning the objectName if it exists. Can you clarify that in the function comment?

Reply:
Done